en/docs/tutorials/connect-your-existing-ballerina-project-to-choreo.md

```diff
@@ -0,0 +1,46 @@
+# Connect Your Existing Ballerina Project to Choreo
+
+Choreo allows you to connect your existing Ballerina project repository to Choreo when you create any of the following components:
```

Comment:
Suggestion:
"You can connect your existing Ballerina project repository to Choreo when you create any of the following components:"

This is to avoid the repetition of "Choreo".

Reply:
Updated as suggested.